wallet-new/integration/Main.hs

```diff
@@ -245,6 +271,7 @@ deterministicTests wc = do
 
         pure (toAcct, toAddr)
 
+
```

Comment:
style nit -- I'd prefer to keep only a line separating functions in a `where` to make it more visually apparent that they're related.

Reply:
ups, that was a mistake. Thanks